challenges/challenge-1/challenge-1.css

```diff
@@ -217,3 +217,19 @@ Saurav Kumar's styling
     transform: translateY(0px);
   }
 }
+/* rzl */
+#rzl{
+  padding: 16px 24px;
+  background: #e8e8f7;
+  border: unset;
+  box-shadow: unset;
+  transition: 0.25s ease-in;
+}
+#rzl img{
+  margin: 16px 0;
+}
+#rzl:hover{
```

Comment:
The animation looks great and was a good design choice to pair with the cleaner look of your cell.

Reply:
Thanks, I just like clean minimalist design